client/express-checkout/blocks/index.js

```diff
@@ -11,7 +11,7 @@ import { getConfig } from 'wcpay/utils/checkout';
 import ExpressCheckoutContainer from './components/express-checkout-container';
 import { checkPaymentMethodIsAvailable } from '../utils/checkPaymentMethodIsAvailable';
 
-const expressCheckoutElementApplePay = ( api ) => ( {
```

Comment:
All the changes here are coming from `client/tokenized-express-checkout/blocks/index.js`